packages/tags/src/Tag.ts

```diff
@@ -71,30 +71,45 @@ export class Tag extends SizedMixin(SpectrumElement, {
     };
 
     private handleKeydown = (event: KeyboardEvent): void => {
-        if (!this.deletable) {
+        if (!this.deletable || this.disabled) {
             return;
         }
         const { code } = event;
+
         switch (code) {
             case 'Backspace':
             case 'Space':
             case 'Delete':
                 this.delete();
-                return;
             default:
                 return;
         }
     };
 
+    private shiftFocusOnEvent(): void {
+        const event = new CustomEvent('sp-tag-focus', {
+            bubbles: true,
+            composed: true,
+            cancelable: true,
+        });
+        this.dispatchEvent(event);
+    }
```

Comment:
This tightly binds the workflow to Tag/Tags, is there a more generic observation method that would apply to all consumers of FocusGroup and RovingTabIndex?

Reply:
Right so do you want this method to go inside RovingTabIndex so that we can leverage it as `this.rovingTabIndexController.shiftFocus()`.